include/deal.II/distributed/fully_distributed_tria_util.h

```diff
@@ -0,0 +1,68 @@
+// ---------------------------------------------------------------------
+//
+// Copyright (C) 2019 by the deal.II authors
+//
+// This file is part of the deal.II library.
+//
+// The deal.II library is free software; you can use it, redistribute
+// it, and/or modify it under the terms of the GNU Lesser General
+// Public License as published by the Free Software Foundation; either
+// version 2.1 of the License, or (at your option) any later version.
+// The full text of the license can be found in the file LICENSE.md at
+// the top level directory of deal.II.
+//
+// ---------------------------------------------------------------------
+
+#ifndef dealii_fully_distributed_tria_util_h
+#define dealii_fully_distributed_tria_util_h
+
+#include <deal.II/distributed/fully_distributed_tria.h>
+
+DEAL_II_NAMESPACE_OPEN
+
+namespace parallel
+{
+  namespace fullydistributed
+  {
+    namespace Utilities
+    {
+      /**
+       * Construct parallel::fullydistributed::ConstructionData from a given
+       * partitioned triangulation tria and a specified process.
+       * The input triangulation can be either
+       * a serial triangulation of type dealii::Triangulation which has been
+       * colored (subdomain_id and/or level_subdomain_id has been set) or a
+       * distributed triangulation of type dealii::distributed::Triangulation.
+       *
+       * @param tria Partitioned input triangulation.
+       * @param tria_pft A parallel::fullydistributed::Triangulation to be
+       *                 passed to extract the settings.
+       * @param my_rank_in Construct ConstructionData for this rank (only
+       *                   working for serial triangulations).
+       * @return ConstructionData to be used to setup
+       *         parallel::fullydistributed::Triangulation.
+       *
+       * @note Multilevel hierarchies are supported if it is enabled in
+       *       parallel::fullydistributed::Triangulation.
+       *
+       * @note Hanging nodes are supported. However, to be able to use this
+       *       feature, the user has to enable multilevel hierarchy support in
+       *       parallel::fullydistributed::Triangulation..
+       *
+       * @author Peter Munch, 2019
+       */
+      template <int dim, int spacedim = dim>
+      ConstructionData<dim, spacedim>
+      copy_from_triangulation(
```

Comment:
Maybe call it `create_construction_data_from_triangulation`? A function called "copy" will generally have an input and output argument, but this one only has input arguments.